web/src/pages/CreateOrphanage.tsx

```diff
@@ -142,10 +166,9 @@ export default function CreateOrphanage() {
 
               <input
                 type="file"
-                id="image[]"
                 multiple
-                accept=".png, .jpg, .jpeg"
```

Comment:
Do not remove the `accept` property, it is used for image type limiting

Reply:
I got it, very good. 

I made a new commit with the change. 

Thanks Matheus 😄